setup.py

```diff
@@ -251,6 +251,31 @@ def version_tag():
     return version
 
 
+def get_available_memory_gb():
```

Comment:
nitpick, maybe this one should go to `tools` instead?

Reply:
moved